config/webpack/loaders/devSiteExample.js

```diff
@@ -26,9 +52,26 @@ const loader = async function loader() {
         isExpanded={isExpanded}
       />
     );`,
-  ].join('\n');
+  ];
+
+  if (cssFileName !== undefined) {
+    exampleCssSource = path.join(parsedResourcePath.dir, cssFileName);
+    code.splice(2, 0, `import Css from '${exampleCssSource}?dev-site-codeblock';`);
+    code.pop();
+    code.push(`export default ({ title, description, isCssExpanded, isExpanded }) => (
```

Comment:
instead of slicing/popping/pushing on the code list, can we break up the current code list, push the scss pieces on the list if needed?
```js
code = [
   ...
   'import ExampleTemplate from \'terra-dev-site/lib/loader-components/_ExampleTemplate\';'
];

if (cssFileName) {
   code.push(`import Css from '${exampleCssSource}?dev-site-codeblock';`)
}

... etc
```

Reply:
I redid this by pushing different versions of the second half of the code based on whether there was a css file or not : [a74dc9a](https://github.com/cerner/terra-dev-site/pull/281/commits/a74dc9a3d2b15e5aa59680d0ff59033e916a0f89).